projects/plugin/src/plugin/plugin.module.ts

```diff
@@ -65,9 +70,8 @@ const components = [
 
 const directives = [HideKeyboardEnterDirective];
 
-
 @NgModule({
-  imports: [CommonModule, FormsModule, IonicModule.forRoot(), TranslateModule.forRoot(), ClipboardModule],
+  imports: [CommonModule, FormsModule, IonicModule.forRoot(), TranslateModule.forRoot(), ClipboardModule, HttpClientModule],
```

Comment:
Don't use the HttpClientModule, wako has its own [http client ](https://github.com/wako-app/mobile-sdk/blob/master/src/services/http/wako-http.service.ts)which uses under the hood mobile http client.